README.md

```diff
@@ -83,6 +83,52 @@ To fire an opened ligtbox event, simply call the `openedLightbox` method of the
 photo.openedLightbox(context);
 ```
 
+### Ecommerce Analytics
+
+For triggering all ecommerce analytics events within your app, you'll want to use the `PXLAnalytics` class. Instantiate one with the application context:
+```
+#!Java
+
+Context c = this.getApplicationContext();
+PXLAnalytics pixleeAnalytics = new PXLAnalytics(c);
+```
+
+#### Add To Cart
+To fire an Add To Cart event, simply call the `addToCart` method of the PXLAnalytics object with the necessary parameters, and an "Add To Cart" event will be fired containing all of the necessary analytics information.
+The parameters for this method are:
+- [Required] sku  (String)
+- [Required] price (String)
+- [Required] quantity (Integer)
+- [Optional] currency (String)
+
+```
+#!java
+
+pixleeAnalytics.addToCart("sku123", "123", 4);
+```
+
+#### Conversion
+To fire a Conversion event, simply call the `conversion` method of the PXLAnalytics object with the necessary parameters, and a "Conversion" event will be fired containing all of the necessary analytics information.
+The parameters for this method are:
+- [Required] cartContents  (ArrayList<HashMap<String, Object>>)
+- [Required] cartTotal (String)
+- [Required] cart_total_quantity (Integer)
+- [Optional] orderId (Integer)
```

Comment:
believe this should be order_id, and can be a string

Reply:
so that's talking abut the variables you pass into the Java function, the SDK takes those and formats them into the json payload correctly: https://github.com/pixlee/android-sdk/pull/9/files#diff-ff24ba1f9b3c78619749436d5d5301dbR59

Reply:
ah ok